### ISSUE TITLE
Duplicative concepts in Visit domain

### ISSUE THREAD

**Issue by m-khitrun** [OHDSI/Vocabulary-v5.0#823]
Two concepts exist in Visit domain that seem to be semantic duplicates: 
[Ambulance Visit](https://athena.ohdsi.org/search-terms/terms/581478) and [Ambulance](https://athena.ohdsi.org/search-terms/terms/38004353)

**Comment by cgreich** [comment]
Not really. "Ambulance" is a descendant of "Ambulance Visit", together with other modes of transportation like own car, train, taxi. We probably don't need those, but they are there. "Ambulance Visit" should be better called "Transportation Visit". But in most cases it is an ambulance.
